menuflow/example-config.yaml

```diff
@@ -1,13 +1,17 @@
 menuflow:
-  database: postgresql://menuflow:secretfoo@postgres/menuflow
-  database_opts:
-      min_size: 1
-      max_size: 10
+    database: postgresql://menuflow:secretfoo@postgres/menuflow
+    database_opts:
+        min_size: 1
+        max_size: 10
 
-  # User to be ignored
-  users_ignore:
-  - '@whatsappbot:example.com'
+    # Ignore messages from this users, it is a list of regex
```

Comment:
The comment should be more specific, maybe say that it accepts user_ids e and regex